style.css

```diff
@@ -413,7 +409,96 @@ body {
   font-weight: normal;
 }
 
-.btn-link {
-  border: 1px solid red;
-  float: left;
+/* contact form */
+
+.form-container {
+  background-color: #6070ff;
+  width: 100%;
+  height: 750px;
+  border: none;
+  left: 0;
+  top: 1rem;
+  border-top-left-radius: 60px;
+}
+
+.form-header {
+  font-size: 40px;
+  font-weight: 800;
+  line-height: 52px;
+  color: #fff;
+  text-align: center;
+  padding-top: 70px;
+}
+
+.form-content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  text-align: center;
+  color: #fff;
+  margin-left: 30px;
+  margin-top: 2rem;
+  max-width: 400px;
+}
+
+li {
+  list-style-type: none;
+}
+
+input {
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  height: 48px;
+  width: 350px;
+  margin-top: 1rem;
+  display: flex;
+  outline: none;
+  gap: 1.5rem;
+  justify-content: center;
+  margin-left: 30px;
+  border-radius: 12px;
+}
```

Comment:
- [ ] Please use the same CSS property from the Figma design on the `input` field

Figma properties eg: 

![fine](https://user-images.githubusercontent.com/79058364/165334253-a57a7401-53bb-4d63-9444-47b40e3c73d5.png)